frontend/src/components/UI/CardSection.vue

```diff
@@ -16,7 +16,7 @@
         <template v-slot:activator="{ on, attrs }">
           <v-btn text :icon="$vuetify.breakpoint.xsOnly" v-bind="attrs" v-on="on" :loading="sortLoading">
             <v-icon :left="!$vuetify.breakpoint.xsOnly">
-              {{ $globals.icons.sort }}
+              {{ $globals.icons.diceMultiple }}
```

Comment:
So you have reverted the change?

Reply:
Not sure what I was doing here. I must not have been paying attention. I'll get it sorted. Sorry about that.

Reply:
No worries, wasn't sure why it was still wrong in the release so I went searching :p